src/topology/algebra/infinite_sum.lean

```diff
@@ -569,6 +606,19 @@ lemma tsum_sum {f : γ → β → α} {s : finset γ} (hf : ∀i∈s, summable (
   ∑'b, ∑ i in s, f i b = ∑ i in s, ∑'b, f i b :=
 (has_sum_sum $ assume i hi, (hf i hi).has_sum).tsum_eq
 
+/-- Version of `tsum_ite_eq_extract` for `add_comm_monoid` rather than `add_comm_group`.
+Requires a different convergence assumption involving `function.update`. -/
+lemma tsum_ite_eq_extract' {f : β → α} (b : β) (hf : summable (f.update b 0)) :
```

Comment:
The name looks strange to me. `tsum_eq_add_tsum_ite`? Same thing twice below.

Reply:
There is already an existing definition `tsum_ite_eq_extract` for groups, would it make sense to also change that naming? I had just copied this naming from that definition.

Reply:
Yes, I'd also change the name for groups, it doesn't make more sense there.

Reply:
Changed these.

There is also some weird naming for the `has_sum` versions. I've changed them to `has_sum_eq_add_has_sum_ite` and `has_sum_ite_sub_has_sum`, but I'm not sure if there's a better name for these, especially since they have somewhat different conclusions.